script/plugin_webpack_directories.rb

```diff
@@ -16,29 +16,61 @@
   end
 end
 
+def log_skip_plugin(plugin_name, reason)
+  STDERR.puts "Skipping plugin #{plugin_name}: #{reason}"
+end
+
+def log_plugin(plugin_name)
+  STDERR.puts "Processing plugin #{plugin_name}"
+end
+
+def entry_name(plugin_name, entry)
+  prefix = File.basename(entry).gsub(/[_]?index.js$/, '')
+  if prefix != ''
+    "#{plugin_name}_#{prefix}"
+  else
+    plugin_name
+  end
+end
+
 config = { entries: {}, paths: [], plugins: {} }
+entry_paths = []
 plugin_name_regexp = /foreman*|katello*/
 specs.each do |dep|
   # skip other rails engines that are not plugins
   # TODO: Consider using the plugin registration api?
   next unless dep.name =~ plugin_name_regexp
-  next if dep.name.include?('_core')
```

Comment:
`next if entry_paths.include?(entry)` on line 65 should handle that case. I consider it better than explicitly excluding `*_core`.

Reply:
I agree hardcoding a name is ugly. Can't remember if this was also done to handle another issue with foreman_ansible_core, but since that package is removed in favor of just foreman_ansible I suspect it won't be an issue. Let's just beware of this in case any issues show up after the merge.

Reply:
this might had a side effect i saw for rex.